<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>587</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RYMCU串口调试助手V1.0  www.rymcu.com</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEditTime">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>370</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>1000</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMS">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>370</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ms</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxPeriodicSend">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>370</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>周期发送</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxComSet">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>161</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>串口设置</string>
    </property>
    <widget class="QPushButton" name="pushButtonOpen">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>160</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>打开串口</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelBaud">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>45</y>
       <width>51</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>波特率</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelStop">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>95</y>
       <width>51</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>停止位</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxCheck">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>125</y>
       <width>69</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Even</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Odd</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mark</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Space</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="labelData">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>51</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>数据位</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxStop">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>99</y>
       <width>69</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1.5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
    </widget>
    <widget class="myComboBox" name="comboBoxNo">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>21</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxData">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>73</y>
       <width>69</width>
       <height>20</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="labelCheck">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>51</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>校验位</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxComBaud">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>47</y>
       <width>69</width>
       <height>20</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <item>
      <property name="text">
       <string>1200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>57600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>74880</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>230400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>460800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>576000</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxSendSet">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>388</y>
      <width>161</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>发送设置</string>
    </property>
    <widget class="QPushButton" name="pushButtonClearSend">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>95</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>清空发送区</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonRdFile">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>95</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>读取文件</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxSendHex">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>120</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>十六进制</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxAddNewShift">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>95</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>发送新行</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxRevSet">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>161</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>接收设置</string>
    </property>
    <widget class="QCheckBox" name="checkBoxRevHex">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>110</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>十六进制</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxReVTime">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>95</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>接收时间</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonClearRev">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>95</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>清空接收区</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonStopRev">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>95</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>停止显示</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSaveRev">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>95</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>保存文件</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxSend">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>388</y>
      <width>541</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>发送区</string>
    </property>
    <widget class="QPlainTextEdit" name="TextSend">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>521</width>
       <height>111</height>
      </rect>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSend">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>110</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxRev">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>341</width>
      <height>361</height>
     </rect>
    </property>
    <property name="title">
     <string>接收区</string>
    </property>
    <widget class="QPlainTextEdit" name="TextRev">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>321</width>
       <height>331</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>370</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>多行发送</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxMutiSend">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>0</y>
      <width>191</width>
      <height>361</height>
     </rect>
    </property>
    <property name="title">
     <string>多行发送区</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>选中</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>输入字符</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditMuti8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>239</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxMuti_4">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>130</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditMuti2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>72</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonMuti1">
     <property name="geometry">
      <rect>
       <x>157</x>
       <y>42</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditMuti10">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>299</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxMuti_7">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>215</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxMuti_8">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>242</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonMuti4">
     <property name="geometry">
      <rect>
       <x>158</x>
       <y>126</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxMuti_6">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>187</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxMuti_2">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>75</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditMuti9">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>269</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>9</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditMuti3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonMuti9">
     <property name="geometry">
      <rect>
       <x>158</x>
       <y>267</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>9</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxMuti_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>272</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditMuti5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>156</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonMuti2">
     <property name="geometry">
      <rect>
       <x>158</x>
       <y>70</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditMuti1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>44</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonMuti6">
     <property name="geometry">
      <rect>
       <x>158</x>
       <y>182</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxMuti_1">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>47</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonMuti5">
     <property name="geometry">
      <rect>
       <x>158</x>
       <y>154</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditMuti6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>184</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonMuti7">
     <property name="geometry">
      <rect>
       <x>158</x>
       <y>210</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxMuti_5">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>159</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditMuti4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>127</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonMuti3">
     <property name="geometry">
      <rect>
       <x>158</x>
       <y>98</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxMuti_3">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>103</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonMuti10">
     <property name="geometry">
      <rect>
       <x>158</x>
       <y>297</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonMuti8">
     <property name="geometry">
      <rect>
       <x>158</x>
       <y>238</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxMuti_10">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>302</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditMuti7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>212</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>330</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="pushButtonMutiReset">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>31</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>复位</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxPeriodicMutiSend">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>0</y>
        <width>131</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>多行循环发送</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
    </widget>
   </widget>
   <zorder>labelMS</zorder>
   <zorder>checkBoxPeriodicSend</zorder>
   <zorder>groupBoxComSet</zorder>
   <zorder>groupBoxSendSet</zorder>
   <zorder>groupBoxRevSet</zorder>
   <zorder>groupBoxSend</zorder>
   <zorder>groupBoxRev</zorder>
   <zorder>radioButton</zorder>
   <zorder>groupBoxMutiSend</zorder>
   <zorder>lineEditTime</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>729</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>myComboBox</class>
   <extends>QComboBox</extends>
   <header>mycombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
